queries.sql

```diff
@@ -0,0 +1,26 @@
+-- animals with the names ending with mon
+SELECT * FROM animals
+WHERE name LIKE '%mon';
+
+-- names of animals date_of_birth BETWEEN '2016-01-01' AND '2019-12-31'
+SELECT name FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+-- names of animals that are neutered and have less than 3 escape attempts
+SELECT name FROM animals
+WHERE neutered = TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals
+WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals
+WHERE weight_kg > 10.5;
+
+SELECT * FROM animals
+WHERE neutered = TRUE;
+
+SELECT * FROM animals
+WHERE name <> 'Gabumon';
+
+SELECT * FROM animals
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] 👍 Good work on providing screenshots for all the queries. However, it's essential to make it clear which query output corresponds to each screenshot. To improve clarity and help reviewers understand the context, please add the screenshots in order or specify which query's output each screenshot represents. This will make it easier for everyone to match the query to its respective result. Keep up the good work! 👏📸📊